code/controllers/subsystem/timer.dm

```diff
@@ -379,6 +379,16 @@ SUBSYSTEM_DEF(timer)
 	next.prev = src
 	prev.next = src
 
+/datum/controller/subsystem/timer/proc/TimerTimeLeft(id)
+	if(!id)
+		CRASH("Bad timer ID!")
```

Comment:
So not passing an ID is a crash, but passing a non-existent timer ID is fine?

Reply:
There is no way currently to know if a given timerID is valid, so crashing on that would just create runtimes for no reason.

Reply:
I might also suggest replacing this was istext() as passing a number would produce a valid timer but is not valid behavior